src/site/content/en/fast/lazy-loading-video/index.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+title: Lazy-loading video
+authors:
+  - jeremywagner
+  - rachelandrew
+date: 2019-08-16
+updated: 2020-06-05
+description: |
+  This post explains lazy-loading and the options available to you when lazy-loading video.
+tags:
+  - performance
+---
+
+As with [image elements](/lazy-loading-images), you can also lazy-load video. When loading video in
+normal circumstances, we do so using the `<video>` element (although [an
+alternate method using
+`<img>`](https://calendar.perfplanet.com/2017/animated-gif-without-the-gif/) has
+emerged with limited implementation). _How_ we lazy-load `<video>` depends on the
+use case, though. Let's discuss a couple of scenarios that each require a
+different solution.
+
+## For video that doesn't autoplay {: #video-no-autoplay }
+
+For videos where playback is initiated by the user (i.e., videos that _don't_
+autoplay), specifying the [`preload`
+attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video#attr-preload)
+on the `<video>` element may be desirable:
+
+```html
+<video controls preload="none" poster="one-does-not-simply-placeholder.jpg">
+  <source src="one-does-not-simply.webm" type="video/webm">
+  <source src="one-does-not-simply.mp4" type="video/mp4">
+</video>
+```
+
+Here, we use a `preload` attribute with a value of `none` to prevent browsers
+from preloading _any_ video data. To occupy the space, we use the `poster`
+attribute to give the `<video>` element a placeholder. The reason for this is
+that default behaviors for loading video can vary from browser to browser:
```

Comment:
```suggestion
The example above uses a `preload` attribute with a value of `none` to prevent browsers
from preloading _any_ video data. The `poster`
attribute gives the `<video>` element a placeholder that will occupy the space while the video loads. The reason for this is
that default behaviors for loading video can vary from browser to browser:
```